src/sentry/api/endpoints/group_ai_summary.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import orjson
+import requests
+from django.conf import settings
+from django.contrib.auth.models import AbstractBaseUser, AnonymousUser
+from pydantic import BaseModel
+from rest_framework.response import Response
+
+from sentry import eventstore, features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.group import GroupEndpoint
+from sentry.api.serializers import EventSerializer, serialize
+from sentry.api.serializers.rest_framework.base import convert_dict_key_case, snake_to_camel_case
+from sentry.models.group import Group
+from sentry.seer.signed_seer_api import sign_with_seer_secret
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
+
+logger = logging.getLogger(__name__)
+
+from rest_framework.request import Request
+
+TIMEOUT_SECONDS = 60 * 30  # 30 minutes
+
+
+class SummarizeIssueResponse(BaseModel):
+    group_id: str
+    summary: str
+    impact: str
+
+
+@region_silo_endpoint
+class GroupAiSummaryEndpoint(GroupEndpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.ML_AI
+    # go away
```

Comment:
Seems a bit rude 🙃

Reply:
Haha it was copied over from some other class a long time ago